README.rst

```diff
@@ -18,7 +18,10 @@
 
 .. image:: https://mybinder.org/badge_logo.svg
  :target: https://mybinder.org/v2/gh/landlab/landlab/release?filepath=notebooks/welcome.ipynb
-
+ 
+ ## Click here to Launch Binder to use the Version 2 Unit Test Interactive Notebook
+.. image:: https://mybinder.org/badge_logo.svg
+ :target: https://mybinder.org/v2/gh/ChristinaB/landlab/ChristinaB-landslide?filepath=notebooks%2Ftutorials%2Flandslides%2FUnit_LandlabLandslide_v2-2020430.ipynb
 =======
```

Comment:
@ChristinaB I'd recommend removing this. The top level readme.rst doesn't really link to component-specific notebooks because there are too many.

Reply:
Done.  Thanks @kbarnhart

Reply:
@ChristinaB it doesn't look like you've removed the links to the landslide notebook here. Maybe I'm missing something.